### PR TITLE
Adding a new template to create N number of VMs with Managed Disks.

### PR DESCRIPTION
**Pull request by sameeraman**
please see the readme file. This template creates N number of VMs with managed disks

### CODE REVIEW

File: 201-vm-copy-managed-disks/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,269 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "VirtualMachineAdminUserName": {
```
Comment (singhkays) [inline 201-vm-copy-managed-disks/azuredeploy.json]: parameters should follow camelCase convention

File: 201-vm-copy-managed-disks/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+        "description": "Globally unique DNS prefix for the Public IPs used to access the Virtual Machines"
+      }
+    }
+  },
+  "variables": {
+    "myVNETName": "myVNET",
+    "myVNETPrefix": "10.0.0.0/16",
+    "myVNETSubnet1Name": "Subnet1",
+    "myVNETSubnet1Prefix": "10.0.0.0/24",
+    "DiagnosticStorageAccountName": "[concat('diagst', uniqueString(resourceGroup().id))]",
```
Comment (singhkays) [inline 201-vm-copy-managed-disks/azuredeploy.json]: camelCase

File: 201-vm-copy-managed-disks/azuredeploy.json (hunk 1 above)
Comment (sameeraman) [inline 201-vm-copy-managed-disks/azuredeploy.json]: Thanks for the feedback. It's fixed now.

File: 201-vm-copy-managed-disks/azuredeploy.json (hunk 2 above)
Comment (sameeraman) [inline 201-vm-copy-managed-disks/azuredeploy.json]: Its's fixed now.

**Comment by singhkays** [comment]
merged

**Pull request merged**